### DISCUSSION TITLE
large number of inferences for purine/pyrimidine metabolism

### DISCUSSION THREAD

**Discussion by ValWood** [monarch-initiative/mondo#4303] [Bigger project]
**MONDO:0019254**

I'm not sure that all of these are errors in purine/pyrimidin metabolism

+ | acute infantile liver failure due to synthesis defect of mtDNA-encoded proteins
-- | --
  | slm3
+ | adenine phosphoribosyltransferase deficiency
  | apt1
+ | adenosine deaminase deficiency
  | dea2
+ | adenylosuccinate lyase deficiency
  | ade8
+ | AICA-ribosiduria
  | ade10
+ | Arts syndrome
  | prs5, SPCC1620.06c
+ | auditory neuropathy-optic atrophy syndrome
  | arh1
+ | autosomal dominant mitochondrial myopathy with exercise intolerance
  | mix17
+ | autosomal recessive ataxia due to ubiquinone deficiency
  | abc1
+ | Bjornstad syndrome
  | bcs1
+ | CAD-CDG
  | arg4
+ | cardioencephalomyopathy, fatal infantile, due to cytochrome c oxidase deficiency 1
  | sco1
+ | cardioencephalomyopathy, fatal infantile, due to cytochrome c oxidase deficiency 2
  | etp1
+ | cardioencephalomyopathy, fatal infantile, due to cytochrome c oxidase deficiency 3
  | coa5
+ | cardioencephalomyopathy, fatal infantile, due to cytochrome c oxidase deficiency 4
  | coa6
+ | cataract-growth hormone deficiency-sensory neuropathy-sensorineural hearing loss-skeletal dysplasia syndrome
  | ism1
+ | Charcot-Marie-Tooth disease recessive intermediate d
  | cox13
+ | Charcot-Marie-Tooth disease type 4K
  | shy1
+ | Charcot-Marie-Tooth disease X-linked recessive 5
  | prs5, SPCC1620.06c
+ | coenzyme Q10 deficiency, primary, 1
  | ppt1
+ | combined oxidative phosphorylation defect type 11
  | sif2
+ | combined oxidative phosphorylation defect type 14
  | msf1
+ | combined oxidative phosphorylation defect type 15
  | fmt1
+ | combined oxidative phosphorylation defect type 17
  | trz2
+ | combined oxidative phosphorylation defect type 2
  | mrps16
+ | combined oxidative phosphorylation defect type 20
  | vrs1
+ | combined oxidative phosphorylation defect type 21
  | SPAC24C9.09
+ | combined oxidative phosphorylation defect type 23
  | mss1
+ | combined oxidative phosphorylation defect type 24
  | slm5
+ | combined oxidative phosphorylation defect type 25
  | msm1
+ | combined oxidative phosphorylation defect type 26
  | trm5
+ | combined oxidative phosphorylation defect type 4
  | tuf1
+ | combined oxidative phosphorylation defect type 7
  | pth3
+ | combined oxidative phosphorylation defect type 8
  | ala1
+ | combined oxidative phosphorylation defect type 9
  | mrpl9
+ | combined oxidative phosphorylation deficiency
  | mef1
+ | combined oxidative phosphorylation deficiency 19
  | isd11, nfs1
+ | combined oxidative phosphorylation deficiency 29
  | trx1
+ | combined oxidative phosphorylation deficiency 33
  | mam33
+ | combined oxidative phosphorylation deficiency 34
  | rsm7
+ | combined oxidative phosphorylation deficiency 35
  | tit1
+ | combined oxidative phosphorylation deficiency 36
  | mrp4
+ | congenital cataract-progressive muscular hypotonia-hearing loss-developmental delay syndrome
  | erv2
+ | cytochrome-c oxidase deficiency disease
  | cfh2, coa3, coa6, cox1, cox10, cox12, cox14, cox2, cox20, cox3, cox6, pet117, sco1, shy1, tac1
+ | deafness-encephaloneuropathy-obesity-valvulopathy syndrome
  | dps1
+ | Diamond-Blackfan anemia
  | rpl1801, rpl35, rps1902
+ | Diamond-Blackfan anemia 1
  | rpl502, rps1901
+ | Diamond-Blackfan anemia 10
  | rps2602
+ | Diamond-Blackfan anemia 11
  | rpl26
+ | Diamond-Blackfan anemia 12
  | rpl15
+ | Diamond-Blackfan anemia 13
  | rps29
+ | Diamond-Blackfan anemia 14 with mandibulofacial dysostosis
  | tsr2
+ | Diamond-Blackfan anemia 15 with mandibulofacial dysostosis
  | rps2602, rps2802, tsr2
+ | diamond-blackfan anemia 16
  | rpl2702
+ | diamond-blackfan anemia 17
  | rps27
+ | Diamond-Blackfan anemia 3
  | rps2402
+ | Diamond-Blackfan anemia 4
  | rps1701
+ | Diamond-Blackfan anemia 5
  | rpl35a
+ | Diamond-Blackfan anemia 6
  | rpl502
+ | Diamond-Blackfan anemia 7
  | rpl1102
+ | Diamond-Blackfan anemia 8
  | rps7
+ | Diamond-Blackfan anemia 9
  | rps1002
+ | encephalopathy due to mitochondrial and peroxisomal fission defect
  | vps1
+ | encephalopathy-hypertrophic cardiomyopathy-renal tubular disease syndrome
  | coq9
+ | erythrocyte AMP deaminase deficiency
  | ada1
+ | familial steroid-resistant nephrotic syndrome with sensorineural deafness
  | coq6
+ | fatal infantile hypertrophic cardiomyopathy due to mitochondrial complex I deficiency
  | SPAC9E9.15
+ | fatal mitochondrial disease due to combined oxidative phosphorylation defect type 3
  | tsf1
+ | glycogen storage disease due to phosphoglycerate kinase 1 deficiency
  | pgk1
+ | glycogen storage disease VII
  | pfk1
+ | gracile syndrome
  | bcs1
+ | growth and developmental delay-hypotonia-vision impairment-lactic acidosis syndrome
  | fsf1
+ | hemolytic anemia due to diphosphoglycerate mutase deficiency
  | gpm1
+ | hemolytic anemia due to glucophosphate isomerase deficiency
  | pgi1
+ | hepatoencephalopathy due to combined oxidative phosphorylation defect type 1
  | SPCC1827.03c
+ | hereditary myopathy with lactic acidosis due to ISCU deficiency
  | isu1
+ | hereditary spastic paraplegia 55
  | pth3
+ | hereditary spastic paraplegia 77
  | msf1
+ | hydrops-lactic acidosis-sideroblastic anemia-multisystemic failure syndrome
  | SPAC4G8.09
+ | hyperinsulinism due to glucokinase deficiency
  | hxk2
+ | hyperuricemia-pulmonary hypertension-renal failure-alkalosis syndrome
  | dia4
+ | hypoxanthine guanine phosphoribosyltransferase partial deficiency
  | hpt1
+ | infantile hypertrophic cardiomyopathy due to MRPL44 deficiency
  | mrpl3
+ | ITPA-related encephalopathy
  | ham1
+ | Leber hereditary optic neuropathy
  | atp6, cob1, cox3
+ | Leigh syndrome with leukodystrophy
  | aim22, coq11, etp1, fmt1, pda1, sdh1, shy1, tac1, SPAC11E3.12, SPBC18E5.10, SPCC417.16
+ | Lesch-Nyhan syndrome
  | hpt1
+ | lethal left ventricular non-compaction-seizures-hypotonia-cataract-developmental delay syndrome
  | oct1
+ | leukoencephalopathy with brain stem and spinal cord involvement-high lactate syndrome
  | dar2
+ | leukoencephalopathy-thalamus and brainstem anomalies-high lactate syndrome
  | mse1
+ | MELAS syndrome
  | cox1, cox2
+ | mitochondrial complex I deficiency
  | SPAC11E3.12, SPAC9E9.15, SPBC18E5.10
+ | mitochondrial complex II deficiency
  | sdh1, sdh2, tim18, SPAC664.12c
+ | mitochondrial complex III deficiency
  | cob1
+ | mitochondrial complex III deficiency nuclear type 1
  | bcs1, qcr7
+ | mitochondrial complex III deficiency nuclear type 3
  | qcr7
+ | mitochondrial complex III deficiency nuclear type 4
  | qcr8
+ | mitochondrial complex III deficiency nuclear type 5
  | qcr2
+ | mitochondrial complex III deficiency nuclear type 6
  | cyt1
+ | mitochondrial complex III deficiency nuclear type 8
  | mzm1
+ | mitochondrial complex III deficiency nuclear type 9
  | cbp4
+ | mitochondrial complex V (ATP synthase) deficiency nuclear type 2
  | SPAC1834.10c
+ | mitochondrial complex V (ATP synthase) deficiency nuclear type 4
  | atp1
+ | mitochondrial complex V (ATP synthase) deficiency, nuclear type 1
  | atp12
+ | mitochondrial DNA deletion syndrome with progressive myopathy
  | dna2
+ | mitochondrial DNA depletion syndrome 1
  | pog1
+ | mitochondrial DNA depletion syndrome 12
  | anc1
+ | mitochondrial DNA depletion syndrome 12a (cardiomyopathic type), autosomal dominant
  | anc1
+ | mitochondrial DNA depletion syndrome 4a
  | pog1
+ | mitochondrial DNA depletion syndrome 4b
  | pog1
+ | mitochondrial DNA depletion syndrome 8a
  | suc22
+ | mitochondrial DNA depletion syndrome 9
  | SPAC16E8.17c
+ | mitochondrial DNA depletion syndrome, encephalomyopathic form with methylmalonic aciduria
  | lsc2
+ | mitochondrial hypertrophic cardiomyopathy with lactic acidosis due to MTO1 deficiency
  | ips1
+ | mitochondrial myopathy with reversible cytochrome C oxidase deficiency
  | slm3
+ | mitochondrial neurogastrointestinal encephalomyopathy
  | pog1, suc22
+ | mitochondrial non-syndromic sensorineural deafness
  | cox1
+ | mitochondrial non-syndromic sensorineural deafness with susceptibility to aminoglycoside exposure
  | cox1, slm3
+ | mitochondrial proton-transporting ATP synthase complex deficiency
  | atp6, atp8
+ | multiple mitochondrial dysfunctions syndrome 1
  | nfu1
+ | multiple mitochondrial dysfunctions syndrome 2
  | fra3
+ | multiple mitochondrial dysfunctions syndrome 3
  | iba57
+ | multiple mitochondrial dysfunctions syndrome 4
  | isa2
+ | multiple mitochondrial dysfunctions syndrome 5
  | isa1
+ | multiple mitochondrial dysfunctions syndrome 6
  | qcr1
+ | myopathy due to myoadenylate deaminase deficiency
  | ada1
+ | myopathy, lactic acidosis, and sideroblastic anemia 1
  | pus3, SPCC576.06c
+ | myopathy, lactic acidosis, and sideroblastic anemia 2
  | SPCC576.06c
+ | myopathy, lactic acidosis, and sideroblastic anemia 3
  | atp6
+ | NARP syndrome
  | atp6
+ | Navajo neurohepatopathy
  | sym1, SPAC3G6.05
+ | neonatal encephalomyopathy-cardiomyopathy-respiratory distress syndrome
  | coq4
+ | neonatal severe cardiopulmonary failure due to mitochondrial methylation defect
  | pet802
+ | optic atrophy 3
  | opa3
+ | orotic aciduria
  | ura5
+ | pancreatic insufficiency-anemia-hyperostosis syndrome
  | cox5
+ | periodic paralysis with later-onset distal motor neuropathy
  | atp6, atp8
+ | Perrault syndrome 2
  | hrs1
+ | Perrault syndrome 4
  | SPAC4G8.09
+ | phosphoribosylpyrophosphate synthetase superactivity
  | SPCC1620.06c
+ | pontocerebellar hypoplasia type 6
  | mrs1
+ | primary coenzyme Q10 deficiency 8
  | coq7
+ | progressive external ophthalmoplegia with mitochondrial dna deletions, autosomal dominant 1
  | pog1
+ | progressive external ophthalmoplegia with mitochondrial DNA deletions, autosomal dominant 2
  | anc1
+ | progressive external ophthalmoplegia with mitochondrial DNA deletions, autosomal dominant 5
  | suc22
+ | progressive external ophthalmoplegia with mitochondrial DNA deletions, autosomal recessive 1
  | pog1
+ | progressive external ophthalmoplegia with mitochondrial DNA deletions, autosomal recessive 2
  | rnh1, suc22
+ | purine nucleoside phosphorylase deficiency
  | SPAC1805.16c
+ | pyruvate kinase deficiency of red cells
  | pyk1
+ | Sengers syndrome
  | anc1
+ | severe X-linked mitochondrial encephalomyopathy
  | aif1
+ | spastic ataxia 3
  | msm1
+ | spastic ataxia 4
  | cid11
+ | spastic ataxia 5
  | yta12
+ | spinocerebellar ataxia type 28
  | yta12
+ | spinocerebellar ataxia with epilepsy
  | pog1
+ | triosephosphate isomerase deficiency
  | tpi1
+ | X-linked intellectual disability-limb spasticity-retinal dystrophy-diabetes insipidus syndrome
  | SPCC1620.06c

**Comment by ValWood** [comment]
I don't have time to go through them all but for example

cardioencephalomyopathy, fatal infantile, due to cytochrome c oxidase deficiency 1 (MONDO:0011451)

glycogen storage diseases look odd too

and some of the aneamias, for example
 hemolytic anemia due to glucophosphate isomerase deficiency

I can see where this comes from
for example glycolysis will make ATP from ADP, but  would people consider this to be an error in purine/pyrimidine metabolism?
I would expect these to be restricted to the pathways of purine and pyrimidine metabolism?

**Comment by ValWood** [comment]
We just mapped from DO to Mondo and the number of purine/pyrimidine disorders  for fission yeast has gone up from 3->152

**Comment by nicolevasilevsky** [comment]
Can anyone weigh in on this? 

@cmungall @pnrobinson 

I'm not sure how to address this.

Here is the explanation for cardioencephalomyopathy, fatal infantile, due to cytochrome c oxidase deficiency 1 (MONDO:0011451)

![image](https://user-images.githubusercontent.com/6722114/82508369-b4225e80-9ab9-11ea-95b8-641bdf2181e9.png)

**Comment by ValWood** [comment]
Actually, although it seems a bit odd  to me I think it's oK to live with the  "oxidative phosphorylation " ones as this could be classed as a defect in "purine or pyrimidine metabolism" although it's really the 'interconversion of the pool or ATP and ADP'. This is probably really a GO issue and one not yet fully resolved.

So the one that stands out for me otherwise is
 Diamond-Blackfan anemia.
Most causes of this are cytosolic ribosome defects  so any issue with pyrimidine metabolism seems more of a symptom than a cause.

**Comment by ValWood** [comment]
Although maybe you could remove the relation between

mitochondrial oxidative phosphorylation disorder (MONDO:0016387)

and

MONDO:0019254
Disease association
inborn disorder of purine or pyrimidine metabolism (MONDO:0019254)

because for many of these the  look quite strange

MONDO:0054865
Disease association
encephalopathy due to mitochondrial and peroxisomal fission defect
for example, is due to defects in  GTPase motors for mitochondrial fission and fusion
is classed as a disease of pyrimidine metabolism via
 mitochondrial oxidative phosphorylation disorder with no known mechanism

**Comment by nicolevasilevsky** [comment]
![image](https://user-images.githubusercontent.com/6722114/87812250-bc2a2080-c814-11ea-8e6c-0da39e60321a.png)

**Comment by ValWood** [comment]
@cmungall  @pnrobinson 
 I get
https://www.pombase.org/term_genes/MONDO:0019254
it seems wrong to have
cytosolic ribosomal proteins and some others described as "disorders of pyrimidine/pyrimidine metabolism"
(I'm pretty sure that the ortholog calls are all correct)

**Comment by ValWood** [comment]
so for example
Diamond-Blackfan anaemia (MONDO:0015253)  (which is caused by mutations in many cytosolic ribosomal proteins)
somehow maps to "disorders of pyrimidine/pyrimidine metabolism"
Maybe that is correct but it was a red flag to me based on my knowledge....

**Comment by pnrobinson** [comment]
@ValWood  I agree
@cmungall @nicolevasilevsky  This is an authoratative source on metabolic disorders. I would suggest that we largely adopt this hierarchy

http://evm.health2media.com/#/detail-view/disorder/0/4
